SoftDevSix/ArgosApplication
# refactor: add path file required project id and file path

## Related Tickets & US
- [CS-43-18](https://app.clickup.com/t/868b5427h)

## Description
The endpoint should receive the project ID and the project path in order to return the adequate coverage file.

GET coverage/project/{projectId}/file?path=src/org/example/AppTest 

Also, update the [documentation](https://github.com/SoftDevSix/ArgosApplication/wiki/Endpoints) of the endpoint.


## Type of change


## How Has This Been Tested?

- Run a project in file manager and get the project id.

- Enter the file path and project id example: http://localhost:8081/coverage/project/06781212-69c1-46d2-b73b-2f69d83e1edc/?filePath=test-jarasm%2Fsrc%2Fmain%2Fjava%2Forg%2Fexample%2FMultiplication.java .

- and see results.

![image](https://github.com/user-attachments/assets/440afeb3-43b9-47bf-a857-d3e0d86338a9)


## Checklist:



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/SoftDevSix/ArgosApplication/39)
